# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/l10n/Localization.h
# Commit: 85194e5b7a9e
# Full Hash: 85194e5b7a9e6ed9114ee46aef0b5efb819cca9b
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 181487bdf6c0 -r 85194e5b7a9e intl/l10n/Localization.h
--- a/intl/l10n/Localization.h	Tue Aug 03 16:25:13 2021 +0000
+++ b/intl/l10n/Localization.h	Tue Aug 03 16:25:13 2021 +0000
@@ -71,10 +71,10 @@
       const dom::Sequence<dom::OwningUTF8StringOrL10nIdArgs>& aKeys,
       nsTArray<dom::Nullable<dom::L10nMessage>>& aRetVal, ErrorResult& aRv);
 
-  void AddResourceId(const nsAString& aResourceId);
-  uint32_t RemoveResourceId(const nsAString& aResourceId);
-  void AddResourceIds(const nsTArray<nsString>& aResourceIds);
-  uint32_t RemoveResourceIds(const nsTArray<nsString>& aResourceIds);
+  void AddResourceId(const nsACString& aResourceId);
+  uint32_t RemoveResourceId(const nsACString& aResourceId);
+  void AddResourceIds(const nsTArray<nsCString>& aResourceIds);
+  uint32_t RemoveResourceIds(const nsTArray<nsCString>& aResourceIds);
 
   void SetAsync();
   bool IsSync();